php-80/Dockerfile

```diff
@@ -80,7 +80,7 @@ ENV VERSION_ZLIB=1.3
 ENV ZLIB_BUILD_DIR=${BUILD_DIR}/zlib
 RUN set -xe; \
     mkdir -p ${ZLIB_BUILD_DIR}; \
-    curl -Ls https://zlib.net/zlib-${VERSION_ZLIB}.tar.xz \
+    curl -Ls http://zlib.net/zlib-${VERSION_ZLIB}.tar.xz \
```

Comment:
We could try `https://github.com/madler/zlib/releases/download/v${VERSION_ZLIB}/zlib-${VERSION_ZLIB}.tar.gz` instead?